src/simplewallet/simplewallet.cpp

```diff
@@ -4840,6 +4840,18 @@ bool simple_wallet::sweep_main(uint64_t below, const std::vector<std::string> &a
     return true;
   }
 
+  size_t outputs = 1;
+  if (local_args.size() > 0 && local_args[0].substr(0, 8) == "outputs=") {
+    if (!epee::string_tools::get_xtype_from_string(outputs, local_args[0].substr(8))) {
```

Comment:
I think it's more reasonable to error out in case of numeric parsing error than to silently ignore it.

Reply:
Done.